data/json/recipes/other/tool.json

```
@@ -146,6 +146,17 @@
     "qualities": [ { "id": "CHISEL", "level": 3 } ],
     "tools": [ [ [ "swage", -1 ] ] ]
   },
+  {
+    "result": "knife_butchering_kit",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+	"reversible": true,
+    "time": "1 s",
```

```suggestion
    "skill_used": "fabrication",
	  "reversible": true,
    "time": "1 s",
```
Should have equal indent